crates/tako/src/internal/server/core.rs

```diff
@@ -280,6 +286,7 @@ impl Core {
     /// it up.
     #[must_use]
     pub fn remove_task(&mut self, task_id: TaskId) -> TaskRuntimeState {
+        self.crash_counter.remove(&task_id);
```

Comment:
Since this is in tako now, can't we just store the counter inside the `Task`?

Reply:
It adds an additional hash map lookup after each task finishes though.. I would rather put it into `Task` and then optimize the size of `Task` more holistically somehow, I'll send a PR for that.

Reply:
Ok, but this map is actually almost always empty, I would consider it as fast op.

Reply:
crash counter was internalized